backtrace-library/src/androidTest/java/backtraceio/library/BacktraceCredentialsTest.java

```diff
@@ -76,4 +78,16 @@ public void createBacktraceCredentialsWithLegacyUriAndGetSubmissionToken() {
         BacktraceCredentials backtraceCredentials = new BacktraceCredentials(Uri.parse(legacyUrl));
         assertEquals(fakeToken, backtraceCredentials.getSubmissionToken());
     }
+
+    @Test
+    public void generateEmptyTokenForInvalidSubmissionUrl() {
+        BacktraceCredentials backtraceCredentials = new BacktraceCredentials("https://submit.backtrace.io/");
+        assertTrue(backtraceCredentials.getSubmissionToken() == null);
```

Comment:
you can use assertNull here and below

Reply:
Thank you - too many quick iterations on this.